airflow/models/baseoperator.py

```diff
@@ -765,6 +765,7 @@ def __init__(
         resources: dict[str, Any] | None = None,
         run_as_user: str | None = None,
         task_concurrency: int | None = None,
+        map_index_template: str | None = None,
```

Comment:
Maybe `map_name_template` is better…? The index is still always an int, the rendered name is a different thing. Not sure about this.

Reply:
what about `map_index_name_template`? It's still an "index", it's just a string instead of an integer.

Reply:
I don’t have a particularly good idea here. Let’s stick with this for now.